.github/workflows/mega-linter.yml

```
@@ -43,7 +43,7 @@ jobs:
           VALIDATE_ALL_CODEBASE: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }} # Validates all source when push on main, else just the git diff with main. Override with true if you always want to lint all sources
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           # ADD YOUR CUSTOM ENV VARIABLES HERE OR DEFINE THEM IN A FILE .mega-linter.yml AT THE ROOT OF YOUR REPOSITORY
-          DISABLE: SPELL # Uncomment to disable copy-paste and spell checks
+          DISABLE: SPELL, MARKDOWN_MARKDOWN_LINK_CHECK # Uncomment to disable copy-paste and spell checks
```

This should instead be: 

```suggestion
          DISABLE: SPELL # Uncomment to disable copy-paste and spell checks
          DISABLE_LINTERS: MARKDOWN_MARKDOWN_LINK_CHECK
```

Please [read the docs](https://megalinter.github.io/latest/configuration/#activation-and-deactivation).